core/scripts/sql/texera_ddl.sql

```diff
@@ -110,6 +110,7 @@ CREATE TABLE IF NOT EXISTS user_project
 (
     `pid`             INT UNSIGNED AUTO_INCREMENT      NOT NULL,
     `name`            VARCHAR(128)                     NOT NULL,
+    `description`     VARCHAR(500),
```

Comment:
if you are changing the ddl to add a column, please provide a sql file to update the existing db instances.

Reply:
Added a script to update the `user_project` table here in `core/scripts/sql/update/projects.sql`